macros/upload_results/upload_results.sql

```diff
@@ -19,7 +19,7 @@
             {% set objects = dbt_artifacts.get_dataset_content(dataset) %}
 
             {# Upload in chunks to reduce query size #}
```

Comment:
```suggestion
            {# Upload in chunks to reduce the query size #}
```
Add to try and re-trigger the approvals